json_journal/preconditioning.py

```diff
@@ -0,0 +1,46 @@
+'''Data sorted before being used by analysis'''
+
+
+def sort_journal_by_date(ledger):
+    '''
+    Sort JSON file by date.
+
+    Parameters
+    ----------
+    Ledger : dict
+        Loaded json file.
+
+    Returns
+    -------
+    Sorted_ledger : dict
+        The same json file, ordered by date.
+    '''
+    return sorted(ledger, key=lambda d: d['date'])
+
+
+def check_entries_sum(ledger):
```

Comment:
Could this function name better represent its purpose?

Reply:
done